tools/sifgen/genesis/genesis.go

```diff
@@ -144,6 +145,44 @@ func InitializeCLP(nodeHomeDir, clpConfigURL string) error {
 	return nil
 }
 
+func InitializeEthbridge(nodeHomeDir, ethbridgeConfigURL string) error {
+	genesis, err := readGenesis(nodeHomeDir)
+	if err != nil {
+		return err
+	}
+
+	response, err := http.Get(fmt.Sprintf("%v", ethbridgeConfigURL))
```

Comment:
We shouldn't be connecting to the network to get this.

Reply:
> > @AustinoBombino Can you explain more of the intention behind this? Sifgen was only ever designed to be something that could allow us to spin up new networks/nodes quickly.
> 
> @intl-man The integration tests were failing prior to adding the new genesis info to sifgen, so I added it in an attempt to fix that.
> But the `InitializeEthbridge()` function was just an attempt to match the existing CLP logic. If you're saying that it isn't necessary we can definitely remove it.
> 
> Can you explain the purpose of the existing `InitializeCLP()` function?

So, that was just a temp solution when we needed to import existing pools when testing the `changePoolFormula` release, back in March. It's no longer needed and should be removed.